lib/src/main/java/grpcbridge/util/ProtoJson.java

```diff
@@ -42,8 +67,17 @@ public static RpcMessage parse(HttpResponse httpResponse, Message.Builder builde
     }
 
     public static HttpResponse serialize(RpcMessage message) {
-        String body = serialize(message.getBody());
-        return new HttpResponse(body, message.getMetadata());
+        String httpBody;
+        if (message.getMethodType().serverSendsOneMessage()) {
```

Comment:
I'd like to see if this can be generalized... Could be exactly the same code but you need to add [] around the body if there is > 1 message, right?

Reply:
yes but what would you do for 1 message or 0 messages, it is a valid case for an endpoint that returns an array but only happens to return 1 message. Using the Method Descriptor tells you whether or not it should be wrapped with []